### ISSUE TITLE
[Suggestion] Transparent scoreboard

### ISSUE THREAD

**Issue by haxballer** [haxball/haxball-issues#170]
Hey, I recently set up various zooms and other items. In my opinion, the transparent scoreboard looks best because at larger resolutions it doesn't cover the part of the image that is important at all. Every time I join HaxBall, I have to manually change it. My question is it possible to set up a transparent scoreboard in the future without having to edit manually code? I know it's nothing but great for people who like higher screen resolutions.

Here you can see: https://images81.fotosik.pl/913/bbe89a6f4ecf51a8.png

Regards

**Comment by awareofdistractions** [comment]
Great idea.
Chatbox could also be transparent:
![screenshot from 2017-11-22 19 52 31](https://user-images.githubusercontent.com/12998823/33144852-bcc11c2a-cfbe-11e7-92b9-4602d335a085.png)
(fullscreen preview)

**Comment by haxballer** [comment]
I agree, mate. I think it's better and it doesn't bother in the game.

**Comment by Leo-MapCreator** [comment]
I agree with the transparent scoreboard, but i dont mind the chat being normal

**Comment by haxballer** [comment]
Chat is fine, but it also covers a large area of the pitch. I hope that in the future it will be possible to minimize the chat. Probably it's the best solution.
